src/app/settings/user/template.html

```diff
@@ -98,6 +94,27 @@
         <km-spinner-with-confirmation [isSaved]="isEqual(settings.selectedProjectID, apiSettings.selectedProjectID)"
                                       fxFlexAlign=" center"></km-spinner-with-confirmation>
       </div>
+      <div fxLayout="row">
+        <div class="entry-label"
+             fxLayoutAlign=" center">Project Landing Page
+        </div>
+        <mat-button-toggle-group [(value)]="selectedProjectLandingPage"
+                                 (change)="changingLandingPage($event.value)"
```

Comment:
can we rename to 
```suggestion
  (change)=" onLandingPageChange($event.value)`
```

e.g we have
- onSettingsChange
- onThemeChange similarly as above